action.yml

```
@@ -24,7 +24,8 @@ runs:
     - name: run updatesnapyaml
       id: updatesnapyaml
       run: |
-        ./desktop-snaps/updatesnap/updatesnapyaml.py --github-user $GITHUB_USER --github-token $GITHUB_TOKEN https://github.com/${{ github.repository }} > snapcraft.yaml
+        ./desktop-snaps/updatesnap/updatesnapyaml.py --github-user $GITHUB_USER --github-token $GITHUB_TOKEN https://github.com/${{ github.repository }}
+        mv output_file snapcraft.yaml
```

If output_file doesn't exist, mv will fail. You should do:

```suggestion
-         mv output_file snapcraft.yaml
+        if [ -f output_file ]; then
+            mv output_file snapcraft.yaml
+        fi
```

---

oh yep, this is needed :)